linera-ethereum/README.md

```diff
@@ -0,0 +1,13 @@
+<!-- cargo-rdme start -->
+
+This module provides functionalities for accessing an ethereum blockchain node.
```

Comment:
We should spell "Ethereum" uppercase everywhere in plain text.

Reply:
Ok, corrected.